doc/radosgw/multisite.rst

```diff
@@ -551,7 +551,7 @@ Starting the radosgw
 --------------------
 
 On the node that is hosting the radosgw of the secondary zone, you would start
-it, similar to the `Starting the Radosgw`_, changing the ``rgw zone=us-east-2``
+it, similar to the `Starting the radosgw`_, changing the ``rgw zone=us-east-2``
```

Comment:
And @tchaikov  I would be most appreciated if you may kindly tell the right way to compile these docs. I know you are so good at compiling and building and it will be a big challenge for me if I have to figure it out by myself. Thanks a lot:)))